packages/devextreme/testing/tests/DevExpress.viz.sparklines/sparkline.tests.js

```diff
@@ -1433,6 +1433,26 @@ QUnit.begin(function() {
         ]);
     });
 
+    QUnit.test('Change winloss lossColor property', function(assert) {
```

Comment:
please make a test name according to our convention, such as 'lossColor property should be able to change at runtime.'